src/views/dataView.component.test.tsx

```diff
@@ -59,14 +59,14 @@ describe('Data View', () => {
 
     await user.click(screen.getByRole('button', { name: 'Filters' }));
 
-    const dialogue = await screen.findByRole('dialog', { name: 'Filters' });
+    const dialogue = await screen.findByRole('dialog');
```

Comment:
Why has this selector become less specific? The dialogue should have a name for a11y purposes and it makes testing like this easier...

Reply:
This shouldn't have been changed, I'll change it back.